README.md

```diff
@@ -2096,6 +2096,31 @@ val a: Unit < Routes =
 
 For further examples, Kyo's [example ledger service](https://github.com/getkyo/kyo/tree/main/kyo-examples/jvm/src/main/scala/kyo/examples/ledger) provides practical applications of these concepts.
 
+### ZIOs: Integration with ZIO
+
+The `ZIOs` effect provides seamless integration between Kyo and the ZIO library. The effect is designed to enable gradual adoption of Kyo within a ZIO codebase. It allows you to use ZIO effects within Kyo computations and vice-versa, leveraging the power and expressiveness of both solutions.
+
+```scala
+import kyo._
+import zio._
+
+// Use the 'get' method to extract a 'ZIO' effect
+val a: Int < (Aborts[Throwable] & ZIOs) =
+  ZIOs.get(ZIO.succeed(42))
```

Comment:
If you don't widen this, it should be inferred as

```scala
val a: Int < ZIOs
```
IMO it's preferable to show that the `succeed` is correctly 'observed'